server/src/test/java/org/elasticsearch/reservedstate/service/FileSettingsHealthIndicatorServiceTests.java

```diff
@@ -101,4 +103,28 @@ public void testGreenYellowYellowGreen() {
             healthIndicatorService.calculate(false, null)
         );
     }
+
+    public void testDescriptionIsTruncated() {
+        checkTruncatedDescription(9, "123456789", "123456789");
+        checkTruncatedDescription(8, "123456789", "1234567…");
+        checkTruncatedDescription(1, "12", "…");
```

Comment:
Nit: Should this truncate the whole thing when the length is 1?  I'd expect the prefix of "12" of length 1 to be "1", and a length 0 to be "".